pyk/setup.cfg

```diff
@@ -18,7 +18,8 @@ where = src
 
 [options.entry_points]
 console_scripts =
-    pyk = pyk.__main__:main
+    pyk    = pyk.__main__:main
+    kit.py = pyk.ktool.kit:main
```

Comment:
```suggestion
    pyk    = pyk.__main__:main
    kit    = pyk.ktool.kit:main
```

Reply:
That creates a conflict with the bash `kit` function. We can resolve this in a future PR.

Reply:
Recorded: https://github.com/runtimeverification/k/issues/2801